web/src/main/java/se/inera/intyg/intygstjanst/web/service/dto/citizen/ListCitizenCertificatesRequestDTO.java

```diff
@@ -0,0 +1,21 @@
+package se.inera.intyg.intygstjanst.web.service.dto.citizen;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Collections;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ListCitizenCertificatesRequestDTO {
```

Comment:
I think you could just call this ListCitizenCertificatesRequest (It is a class that is used for communicating with external systems).

Also, because it doesn't need any serialization/deserialization you can just let it be a @Value and @Builder.